packages/editor/src/components/collapsible-block-toolbar/index.js

```diff
@@ -42,6 +42,15 @@ export default function CollapsibleBlockToolbar( { isCollapsed, onToggle } ) {
 		}
 	}, [ blockSelectionStart, onToggle ] );
 
+	// Reset isCollapsed when the collapsible block toolbar unmounts. It is true
+	// by default.
+	useEffect(
+		() => () => {
+			onToggle( true );
```

Comment:
It doesn't feel like the right place to call this for me. It should be the responsibility of the parent component IMO (unless the state is moved to within this component)

Reply:
The thing is that the parent component never sets this state, it appears it's only used inside this one. But the parent component needs to be aware of it to hide the document bar. Tbh, I think the local state is bad here, it should probably be done differently. I just didn't feel like refactoring it.

Reply:
I changed the whole thing to use CSS.